### ISSUE TITLE
method completion based on class ancestors?

### ISSUE THREAD

**Issue by DeeeeLAN** [TheClams/SystemVerilog#2]
When I am auto-completing a task or function from a class, it only shows methods within that class, no inherited methods. Is this something that should be there but I am not seeing it for some reason, or would it need to be added?

**Comment by TheClams** [comment]
It is not implemented yet, mainly for performances reason..

**Comment by DeeeeLAN** [comment]
Makes sense. Could it be put on background threads and the list update as the threads return? 

I think the ideal would be if each class was cached somewhere with all its methods and ancestor methods, then you just have to retrieve that list. I don’t know if you are currently doing much caching or not.
